projects/ui/src/shim.cds-core.scss

```diff
@@ -1826,6 +1826,9 @@
   .clr-vertical-nav.has-icons .nav-group-children .nav-link {
     padding: var(--clr-vertical-nav-item-child-padding);
   }
+  .clr-vertical-nav.has-nav-groups .nav-group-content > .nav-link {
+    font-weight: var(--clr-vertical-nav-item-top-level-font-weight);
+  }
```

Comment:
Is this needed when we have it added inside [_vertical-nav.clarity.scss](https://github.com/vmware-clarity/ng-clarity/pull/1018/files#diff-241bd325c28d64d8afd38964345109e89ed416488824ee6bc9d6625d0a758938R513) 
 and it's pointing to the same property?

Reply:
Good catch!